docs/resources/secondary_connection.md

```diff
@@ -17,20 +17,20 @@ Resource used to manage secondary (replicated) connections. To manage primary co
 ## Minimal
 resource "snowflake_secondary_connection" "basic" {
   name          = "connection_name"
-  as_replica_of = "<organization_name>.<account_name>.<connection_name>"
+  as_replica_of = "\"<organization_name>\".\"<account_name>\".\"<connection_name>\""
 }
 
 ## Complete (with every optional set)
 resource "snowflake_secondary_connection" "complete" {
   name          = "connection_name"
-  as_replica_of = "<organization_name>.<account_name>.<connection_name>"
+  as_replica_of = "\"<organization_name>\".\"<account_name>\".\"<connection_name>\""
```

Comment:
This gives me an idea that we can also export external identifiers like we do with `fully_qualified_name`, wdyt? (ofc, this would be a future improvement)

Reply:
Yes, I already suggested it before. Created SNOW-1852211.